Source/WebCore/platform/graphics/skia/ShareableBitmapSkia.cpp

```diff
@@ -31,9 +31,11 @@
 #include "GraphicsContextSkia.h"
 #include "NotImplemented.h"
 
+WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN // GLib/Win ports
 IGNORE_CLANG_WARNINGS_BEGIN("cast-align")
 #include <skia/core/SkSurface.h>
 IGNORE_CLANG_WARNINGS_END
+WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_END
```

Comment:
```suggestion
WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN // GLib/Win ports
#include <skia/core/SkSurface.h>
WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_END
```

You can remove the cast-align ignore now, since you've added a bigger hammer.

Reply:
I suppose the same applies to the changes in `NativeImageSkia.cpp`.